tests/storage/mpathconf_test.py

```diff
@@ -49,6 +49,21 @@
 }
 """
 
+MPATH_CONF = f"""\
+{mpathconf.CURRENT_TAG}
+# Valid configuration
+"""
+
+MPATH_CONF_PRIVATE = f"""\
+{mpathconf.CURRENT_TAG}
+{mpathconf._PRIVATE_TAG}
```

Comment:
Missing comment, or maybe we don't need the comment in other examples. The constant name can be enough to explain the example.